lms/static/js/learner_dashboard/views/program_card_view.js

```diff
@@ -23,6 +24,25 @@
                 render: function() {
                     var templated = this.tpl(this.model.toJSON());
                     this.$el.html(templated);
+                    this.postRender();
+                },
+                postRender: function() {
+                    if(navigator.userAgent.indexOf('MSIE')!==-1
```

Comment:
Is this still needed for IE10 and IE11 (i.e. were you able to recreate the bug without this code present)? I was hoping that the latest version of pictureFill would have resolved the bug.